src/app/services/grid-utils.service.ts

```diff
@@ -80,4 +80,15 @@ export class GridUtilsService {
   constructor(private router: Router) {
   }
 
+  targetGeneValueFormatter(params: any) {
+
+    if (params.colDef.field === 'target_genes' && params.data.target_genes) {
```

Comment:
The name of the field should not be part of the formatter's code. We would like to keep it generic so it can be used for other fields as well, when needed.

Reply:
Done.